### PR TITLE
ingest: use new augur curate commands

### PR DESCRIPTION
**Pull request by joverlee521**
## Description of proposed changes

Updates the curate rule to use the new augur curate commands. 
Updates `ingest/vendored` to pull in latest changes including [dropping the curation scripts](https://github.com/nextstrain/ingest/pull/45). 

## Related issue(s)

Resolves https://github.com/nextstrain/pathogen-repo-guide/issues/49
Corresponding docs update in https://github.com/nextstrain/docs.nextstrain.org/pull/213

## Checklist

- [ ] Checks pass

### CODE REVIEW

**Comment by joverlee521** [comment]
Comparing outputs from the curate rule from before and after updates. 

Starting from the main branch.

1. Run the ingest workflow on main branch for zika
```
nextstrain build ingest curate --config ncbi_taxon_id='64320'
```
2. Keep a copy of the output
```
mv ingest/data/all_metadata.tsv ingest/data/main_all_metadata.tsv
```
3. Switch to this PR branch
```
git checkout update-curate
```
4. Use my local venv for augur on the master branch to include unreleased [bug fix for `parse-genbank-location`](https://github.com/nextstrain/augur/pull/1531) 
```
pyenv activate augur
```
5. Re-run the curate rule with changes
```
nextstrain build --ambient ingest curate --config ncbi_taxon_id='64320'
```
6. Diff results 
```
daff diff ingest/data/main_all_metadata.tsv ingest/data/all_metadata.tsv > ingest/data/all_metadata_diff.txt
```

Inspected the [diff output]((https://github.com/user-attachments/files/16198291/all_metadata_diff.txt)) and the only change is for the abbreviated authors output from `<first_author> et al` to `<first_author> et al.`

File: ingest/rules/curate.smk [hunk 1]
```diff
@@ -85,30 +86,30 @@ rule curate:
...
                 --default-value {params.authors_default_value} \
                 --abbr-authors-field {params.abbr_authors_field} \
-            | ./vendored/apply-geolocation-rules \
+            | augur curate apply-geolocation-rules \
                 --geolocation-rules {input.all_geolocation_rules} \
-            | ./vendored/merge-user-metadata \
+            | augur curate apply-record-annotations \
                 --annotations {input.annotations} \
                 --id-field {params.annotations_id} \
-            | augur curate passthru \
```
Comment (j23414) [inline ingest/rules/curate.smk:111]: Nice to see that we can now omit "curate passthru", seems more efficient.

My local testing showed that everything functioned as expected. Additionally, I don't mind the removal of the period at the end of "author et al" so looks good to me!

**Review by j23414 — APPROVED** [review]

File: ingest/rules/curate.smk (hunk 1 above)
Comment (joverlee521) [reply ingest/rules/curate.smk:111]: > I don't mind the removal of the period at the end of "author et al" so looks good to me!

The new `augur curate abbreviate-authors` is actually adding the period so it's better!

Comment (j23414) [reply ingest/rules/curate.smk:111]: lol, my brain slipped a bit (true<->false) 😆 Thanks

**Review by genehack — APPROVED** [review]

**Comment by joverlee521** [comment]
With [Augur 25.1.1](https://github.com/nextstrain/augur/releases/tag/25.1.1) released, I updated my Docker runtime and verified the workflow still worked as expected 🎉

**Pull request merged**